FlowerinoPlugin/src/org/flowerplatform/flowerino_plugin/command/UpdateSourceFilesAndCompileCommand.java

```diff
@@ -0,0 +1,221 @@
+package org.flowerplatform.flowerino_plugin.command;
+
+import static org.flowerplatform.flowerino_plugin.FlowerPlatformPlugin.log;
+
+import java.io.File;
+import java.io.IOException;
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+import java.util.ArrayList;
+
+import org.flowerplatform.flowerino_plugin.FlowerPlatformPlugin;
+import org.flowerplatform.flowerino_plugin.SourceFileDto;
+import org.flowerplatform.flowerino_plugin.util.RunnableWithListener;
+import org.flowerplatform.flowerino_plugin.util.StartEndListener;
+import org.flowerplatform.flowerino_plugin.util.Util;
+import org.flowerplatform.tiny_http_server.HttpCommandException;
+import org.flowerplatform.tiny_http_server.IHttpCommand;
+import org.flowerplatform.tiny_http_server.ReflectionException;
+
+import processing.app.BaseNoGui;
+import processing.app.Editor;
+
+/**
+ * Loads the given files into Arduino IDE, and invokes compile on them.
+ * @author Claudiu Matei
+ * @author Andrei Taras
+ */
+public class UpdateSourceFilesAndCompileCommand extends ArrayList<SourceFileDto> implements IHttpCommand {

```

Comment:
As vrea sa spargem in 2 comenzi. Si sa nu extindem acel ArrayList. Sa avem o lista de fisiere.